frontend/src/ui/Blocks/Series.tsx

```diff
@@ -219,6 +219,7 @@ type GridTypeProps = {
 
 const GridTile: React.FC<GridTypeProps> = ({ event, basePath, active }) => {
     const TRANSITION_DURATION = "0.3s";
+    const date = event.isLive ? String(event.syncedData?.startTime) : event.created;
```

Comment:
The `String()` here is incorrect. As that would result in the string `"undefined"` if `syncedData` is undefined. 

```suggestion
    const date = event.isLive ? event.syncedData?.startTime ?? event.created : event.created;
```

This uses the created time as fallback.